### ISSUE TITLE
Immersive engineering sync. When using hammer + ingots tool goes into storage.

### ISSUE THREAD

**Issue by sherkas** [zerofall/EZStorage2#41]
Place IE hammer or cutters in crafting grid of EZStorage. Shiftclick ingots to mass produce X amount of plates or wires. The system will place the tool after crafting back into storage.

dont know if its intended but I think it should be leaving the tool there.

**Comment by sblectric** [comment]
Do you mean the tool goes into storage after you exit the EZStorage GUI? Because that's the intended behavior, as multiple players can use the crafting grid on a system at the same time and not interfere with each other.

**Comment by sherkas** [comment]
No, if you place 10 iron ingots in the crafter and the hammer and right click to do only one (not the whole stack) the hammer goes back into storage. The GUI is still open at this point. The proper behaviour would be making 1 plate. If you shift click to make the whole stack, the effect is the same. The hammer goes into storage (which is mostly okay but still)
I like how when you close the GUI it does that, but its also trigger on craft attempts
